### DISCUSSION TITLE
What is the branch "single_target" used for?

### DISCUSSION THREAD

**Discussion by git4ydh** [ultralytics/yolov5#1792] [Q&A]
There is a branch "single_target", is it for detecting  single class object such as car?  Or what is it for? Thank you

**Comment by glenn-jocher** [comment]
@git4ydh `single_target` branch implements a single-anchor to label matching scheme similar to https://github.com/ultralytics/yolov3. It is for internal testing purposes and is not recommended for use in new projects.
